src/layouts/Base.astro

```diff
@@ -1,5 +1,9 @@
 ---
-import { Navigation } from '@components';
+import { DonationButtons, GetInTouch, Navigation } from '@components';
+import { getDonationOptions } from '../data/stripe';
+
+const { enabled: donationsEnabled, options: donationOptions } =
+	await getDonationOptions();
```

Comment:
🔍 Here, we get our donation information as front matter, and can then pass it into the `DonationOptions` component that relies on it.